skbio/alignment/_tabular_msa.py

```diff
@@ -201,6 +201,16 @@ def keys(self):
 
     @property
     @experimental(as_of='0.4.0-dev')
+    def minter(self):
```

Comment:
`has_keys` tells whether or not there are keys on the MSA, regardless of whether there is a minter. Keys can be set either via `keys` or a minter in the constructor and `reindex`. `has_minter` tells whether or not there is a minter on the MSA.